dot/sync/chain_sync_integration_test.go

```diff
@@ -75,12 +73,14 @@ func TestChainSync_sync_bootstrap_withWorkerError_Integration(t *testing.T) {
 func TestChainSync_sync_tip_Integration(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	cs := newTestChainSync(ctrl)
-	cs.blockState = new(mocks.BlockState)
 	header, err := types.NewHeader(common.Hash{0}, trie.EmptyHash, trie.EmptyHash, 1000,
 		types.NewDigest())
 	require.NoError(t, err)
-	cs.blockState.(*mocks.BlockState).On("BestBlockHeader").Return(header, nil)
-	cs.blockState.(*mocks.BlockState).On("GetHighestFinalisedHeader").Return(header, nil)
```

Comment:
oh god it was not even used 🤯

Reply:
What do you mean?

Reply:
the blockstate mockery mock... well it was never used actually since it's removed now and it passes.